app/src/main/java/com/geeksville/mesh/ui/MapFragment.kt

```diff
@@ -1,680 +1,316 @@
-                    .metadata(Value(STYLE_PACK_METADATA))
-                    .build(),
-                { progress ->
-                    updateStylePackDownloadProgress(
-                        progress.completedResourceCount,
-                        progress.requiredResourceCount,
-                    )
-                },
-                { expected ->
-                    if (expected.isValue) {
-                        expected.value?.let { stylePack ->
-                            // Style pack download finishes successfully
-                            debug("StylePack downloaded: $stylePack")
-                            if (binding.stylePackDownloadProgress.progress == binding.stylePackDownloadProgress.max) {
-                                debug("Style pack download complete")
-                                binding.stylePackText.visibility = View.INVISIBLE
-                                binding.stylePackDownloadProgress.visibility = View.INVISIBLE
-                                stylePackDownloadSuccess = true
-                            } else {
-                                debug("Waiting for tile region download to be finished.")
-                            }
-                        }
-                    }
-                    expected.error?.let {
-                        stylePackDownloadSuccess = false
-                        // Handle error occurred during the style pack download.
-                        binding.stylePackText.visibility = View.INVISIBLE
-                        binding.stylePackDownloadProgress.visibility = View.INVISIBLE
-                        debug("StylePackError: $it")
-                    }
-                }
-            )
-
-            // - - - - - - - -
-
-            // 2. Create a tile region with tiles for the outdoors style
-
-            // A Tile Region represents an identifiable geographic tile region with metadata, consisting of
-            // a set of tiles packs that cover a given area (a polygon). Tile Regions allow caching tiles
-            // packs in an explicit way: By creating a Tile Region, developers can ensure that all tiles in
-            // that region will be downloaded and remain cached until explicitly deleted.
-
-            // Creating a Tile Region requires supplying a description of the area geometry, the tilesets
-            // and zoom ranges of the tiles within the region.
-
-            // The tileset descriptor encapsulates the tile-specific data, such as which tilesets, zoom ranges,
-            // pixel ratio etc. the cached tile packs should have. It is passed to the Tile Store along with
-            // the region area geometry to load a new Tile Region.
-
-            // The OfflineManager is responsible for creating tileset descriptors for the given style and zoom range.
+    override fun onPause() {
+        val edit = mPrefs.edit()
+        edit.putString(prefsTileSource, map.tileProvider.tileSource.name())
```

Comment:
I need to determine if this is actually needed after setting setDestroyMode(false)

Reply:
Determined this wasn't needed